themes/custom/az_barrio/config/install/az_barrio.settings.yml

```diff
@@ -31,15 +34,6 @@ bootstrap_barrio_button: 1
 bootstrap_barrio_button_size: ''
 bootstrap_barrio_button_outline: 0
 
-# Navbar
-# --------------
-bootstrap_barrio_navbar_toggle: 1
-bootstrap_barrio_navbar_container: true
-bootstrap_barrio_navbar_top_position: ''
-bootstrap_barrio_navbar_top_color: 'navbar-light'
-bootstrap_barrio_navbar_position: ''
-bootstrap_barrio_navbar_color: 'navbar-light'
-
```

Comment:
We should probably at least make sure that removing these doesn't break anything.

Per 10/23/2020 meeting discussion from @tadean:
> When these weren't present on theme settings form, it caused unexpected changes to navbar (in previous PRs), e.g. the navbar suddenly becoming container fluid.